src/core/utils.ts

```diff
@@ -62,38 +62,38 @@ export const pluck = function (n, f?) {
  * @type {{}}
  */
 export const utils: IUtils = {
```

Comment:
These utils are confusing - do they still need to be filled in?

Reply:
Oh, I guess they get replaced below. I guess these dummies are a way of giving type information to the methods below?

Reply:
This a temporary structure to get started, this pattern will totally go. In cases where functions are independent, these will be exposed as top-level (with backward compatibility). In other cases when they are related they will be rearranged to avoid this pattern.